### ISSUE TITLE
If i do a SMBC failover to site B then delete the LUN mappings at site A  and remove the protection relationship MPIO breaks 

### ISSUE THREAD

**Issue by Brandon12345678901** [NetAppDocs/ontap#1431]
### Page URL

https://docs.netapp.com/us-en/ontap/snapmirror-active-sync/remove-configuration-task.html

### Page title

Remove a SnapMirror active sync configuration

### Summary

Can we warn customers  not to failover before removing config or not to use SMBC for migration  from site A - Site B  by doing failover ?
Customer used SMBC failed over iscsi LUN to site B then removed Site A LUN mapping and SMBC protection so only presenting at site B , now after windows reboot ...  
MS MPIO / DSM is not working disks are not visible in device manager sees hidden disk with 
"Currently, this hardware device is not connected to the computer. (Code 45)"
, we only see hidden Disk devices and  volumes ....  in iSCSI initiator we see **Disk -1** port x bus X target X LUN ID x 
- If MS MPIO  is removed we can see all devices on each path ie 4 of the same disk.  
I can repo this behavior in the Lab ,I have tried changing LUN ID , Serial, disk signature , checked for SCSI reservation etc..   tried  Hacking windows registry to remove persistent devices from  SCSI and volume devices The only solution i have so far is install Host utilities seems to add extra MPIO device hardware ID’s  seems issue is related to MSFT2005iSCSIBusType_0x9 ... then create vol file clone of the lun then we can see the LUN  again with MPIO  but all paths are active/unoptimized .................... any help or advise would also be much appreciated or if there is someone you know who specializes in Microsoft iSCSI initiator and MPIO / DSM integration  debugging analysis. 
 Windows MPIO event : 
Log Name:      System
Source:        mpio
Date:          7/31/2024 1:48:37 PM
Event ID:      47
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      SQL1.demo.netapp.com
Description:
An error occurred while the system tried to claim a Multipath I/O (MPIO) path. The location of the failing path is described by the SCSI address (4, 0, 1, 21) (Port, Bus, Target, Lun). To resolve this issue, either reboot the system, or show hidden devices in Device Manager and try uninstalling the specific disk device indicated above, and then rescan for hardware changes.
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="mpio" />
    <EventID Qualifiers="49159">47</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2024-07-31T13:48:37.567792200Z" />
    <EventRecordID>26533</EventRecordID>
    <Channel>System</Channel>
    <Computer>SQL1.demo.netapp.com</Computer>
    <Security />
  </System>
  <EventData>
    <Data>\Device\MPIOControl</Data>
    <Data>4, 0, 1, 21</Data>
    <Binary>0000000002002C00000000002F0007C00B000000BB0000C000000000000000000000000000000000</Binary>
  </EventData>
</Event>

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-ahibbard** [comment]
Thanks for reporting this @Brandon12345678901 . We will address as soon as possible. Is there a support case that exists for this issue?

**Comment by Brandon12345678901** [comment]
Hi Andrew,

Thanks for taking a look  the case is NetApp log # 2010107714 Win MPIO DSM

I am out the office next week so looping in my colleagues.

This was the lab I reproduced the issue in

Protecting Critical Workloads with SnapMirror Active Sync v1.0
Lab ID: RT11146582


ReF:
 Page URL
https://docs.netapp.com/us-en/ontap/snapmirror-active-sync/remove-configuration-task.html
Page title
Remove a SnapMirror active sync configuration
Summary
Can we warn customers not to failover before removing config or not to use SMBC for migration from site A - Site B by doing failover ?
Customer used SMBC failed over iscsi LUN to site B then removed Site A LUN mapping and SMBC protection so only presenting at site B , now after windows reboot ...
MS MPIO / DSM is not working disks are not visible in device manager sees hidden disk with
"Currently, this hardware device is not connected to the computer. (Code 45)"
, we only see hidden Disk devices and volumes .... in iSCSI initiator we see Disk -1 port x bus X target X LUN ID x

  *   If MS MPIO is removed we can see all devices on each path ie 4 of the same disk.
I can repo this behaviour in the Lab ,I have tried changing LUN ID , Serial, disk signature , checked for SCSI reservation etc.. tried Hacking windows registry to remove persistent devices from SCSI and volume devices The only solution i have so far is install Host utilities seems to add extra MPIO device hardware ID's seems issue is related to MSFT2005iSCSIBusType_0x9 ... then create vol file clone of the lun then we can see the LUN again with MPIO but all paths are active/unoptimized .................... any help or advise would also be much appreciated or if there is someone you know who specializes in Microsoft iSCSI initiator and MPIO / DSM integration debugging analysis.
Windows MPIO event :
Log Name: System
Source: mpio
Date: 7/31/2024 1:48:37 PM
Event ID: 47
Task Category: None
Level: Error
Keywords: Classic
User: N/A
Computer: SQL1.demo.netapp.com
Description:
An error occurred while the system tried to claim a Multipath I/O (MPIO) path. The location of the failing path is described by the SCSI address (4, 0, 1, 21) (Port, Bus, Target, Lun). To resolve this issue, either reboot the system, or show hidden devices in Device Manager and try uninstalling the specific disk device indicated above, and then rescan for hardware changes.
Event Xml:
47 2 0 0x80000000000000 26533 System SQL1.demo.netapp.com \Device\MPIOControl 4, 0, 1, 21 0000000002002C00000000002F0007C00B000000BB0000C000000000000000000000000000000000

Kind regards,

Brandon Stockil
VCP5-DCV, VCP3, NCTA, MCP
Senior Technical Support Engineer (SAN)

NetApp
+31 20 703 6881 Direct Phone
+31 20 703 6735 Office Phone<https://www.netapp.com/us/contact-us/support.aspx>
***@***.******@***.***>
